src/App.js

```diff
@@ -1,27 +1,32 @@
 import React from "react";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
-import { lazy } from "react";
+import { lazy,Suspense } from "react";
 import * as ROUTES from './constants/routes'
 import Layout from "./Components/Layout/Layout";
+import Loader from "./Components/Loader/Loader";
 const Login = lazy(() => import("./pages/Login/Login"));
 const SignUp = lazy(() => import('./pages/SignUp/SignUp'));
 const Dashboard = lazy(() => import('./pages/Dashboard/Dashboard'));
 const MyProfile = lazy(() => import('./pages/MyProfile/MyProfile'));
 const FriendsProfile = lazy(() => import('./pages/FriendsProfile/FriendsProfile'));
+const Settings = lazy(() => import('./pages/Settings/Settings'));
 
 function App() {
   return (
     <div>
       <BrowserRouter>
+      <Suspense fallback={<Loader fadeOutInterval={5000} />}>
```

Comment:
I remember in the beginning we discussed using this loader and you mentioned it will only be for the login page.
You should not use it with an interval if you are going to use it this way.